proto/text.go

```diff
@@ -594,15 +594,15 @@ func writeString(w *textWriter, s string) error {
 
 func writeUnknownStruct(w *textWriter, data []byte) (err error) {
 	if !w.compact {
-		if _, err := fmt.Fprintf(w, "/* %d unknown bytes */\n", len(data)); err != nil {
+		if _, err := fmt.Fprintf(w, "# %d unknown bytes\n", len(data)); err != nil {
 			return err
 		}
 	}
 	b := NewBuffer(data)
 	for b.index < len(b.buf) {
 		x, err := b.DecodeVarint()
 		if err != nil {
-			_, err := fmt.Fprintf(w, "/* %v */\n", err)
+			_, err := fmt.Fprintf(w, "# %v\n", err)
```

Comment:
I don't think the error message should be written to the output as it is already returned anyways.  I'd prefer removing this type of comment, same for the rest below.